src/mgallocator.c

```diff
@@ -15,12 +15,17 @@
 #include "mgallocator.h"
 
 #include <assert.h>
+// Somehow stdalign.h is not there (VS Build Tools 2019, Windows 11)
+// https://docs.microsoft.com/en-us/cpp/cpp/alignment-cpp-declarations?view=msvc-170
+#if !defined(_WIN32) || !defined(_MSC_VER)
 #include <stdalign.h>
+#endif
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 
 #ifdef _MSC_VER
+#define alignof __alignof
```

Comment:
I think the conditions here should be the same, as both of them should be there or none of them.

Reply:
I've polished a bit, I'll add the CI job and merge when everything is compiling